src/iOS/toga_iOS/libs/core_graphics.py

```diff
@@ -1,6 +1,114 @@
 ##########################################################################
 # System/Library/Frameworks/CoreGraphics.framework
 ##########################################################################
+from ctypes import *
+from ctypes import util
+
+from rubicon.objc.types import register_preferred_encoding
+from rubicon.objc import *
```

Comment:
At column 1: (F403) 'from rubicon.objc import *' used; unable to detect undefined names